app-text/cutemarked/cutemarked-0.11.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A Qt-based Markdown editor"
+DESCRIPTION="CuteMarkEd is a Qt-based Markdown editor with live HTML preview, \
+math expressions, code and markdown syntax highlighting."
+HOMEPAGE="https://cloose.github.io/CuteMarkEd"
+COPYRIGHT="2013 Christian Loose
+	2007 David Loren Parsons
+	2011-2012 Ali Rantakari
+	2012 CodeCatalyst, LLC"
+LICENSE="GNU GPL v2
+	MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/cloose/CuteMarkEd/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="78a41808c2f0452375810abdff76eeaaee012f8d1368a2b8772ec6b4d2ceeec8"
+SOURCE_DIR="CuteMarkEd-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	cutemarked$secondaryArchSuffix = $portVersion
+	app:CuteMarkEd$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libhunspell_1.6$secondaryArchSuffix
+	lib:libmarkdown$secondaryArchSuffix
+	lib:libQt5WebKit$secondaryArchSuffix
```

Comment:
ABC ordering, please.

Reply:
I'd say a non-case-sensitive sorting should be OK.
Would you like a case-sensitive sorting?
(This might not be a good idea imho.)

Reply:
non-case-sensitive sorting only.